apple/Sources/Sargon/Extensions/Methods/Profile/Factor/PasswordFactorSource+Functions.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  File.swift
+//
+//
+//  Created by Alexander Cyon on 2024-06-02.
```

Comment:
Created by Dan on 2024-11-29 :)

Reply:
Oh, it was just a rename, but I'll remove this anyway :)